mathesar/imports/csv.py

```diff
@@ -149,6 +142,10 @@ def create_db_table_from_data_file(data_file, name, schema, comment=None):
         update_pk_sequence_to_latest(engine, table)
     except (IntegrityError, DataError):
         drop_table(name=name, schema=schema.name, engine=engine)
+        column_names_alt = [
+            column_name if column_name != ID else ID_ORIGINAL
+            for column_name in column_names
+        ]
```

Comment:
Didn't make changes here, only moved it since it's only used in this try-except branch.